src/dune_rules/foreign_rules.ml

```diff
@@ -94,7 +94,7 @@ let build_c ~kind ~sctx ~dir ~expander ~include_flags (loc, src, dst) =
   let output_param =
     match ctx.lib_config.ccomp_type with
     | Msvc -> [ Command.Args.Concat ("", [ A "/Fo"; Target dst ]) ]
-    | Other _ -> [ A "-o"; Target dst ]
+    | Other _ -> [ A "-o"; Target dst; A "-fdiagnostics-color=always" ]
```

Comment:
This flag should be defined in the `:standard` set (in `Super_context.default_context_flags`) for it to be user-overriden. Maybe it is worth adding it only is ANSi colors are supporter.

Reply:
Thanks, moved to `Super_context.default_context_flags`.

Reply:
... and now with `Ansi_color.stderr_supports_color`. The compilers print errors on stderr.